backend/server.py

```diff
@@ -260,3 +287,75 @@ def export_translation(source_id: int):
             status_code=500, detail=f"Error generating DOCX: {str(e)}")
 
 
+####### DICTIONARY
+@app.post("/dictionary/{source_id}", response_model=dict)
+def create_dictionary_or_version_for_source(source_id: int, user_info: dict = Depends(get_user_info)):
```

Comment:
You can either here add examples and convert them to rules or later do another call and all example rules to dictionary.